Check_Prime.rb

```diff
@@ -0,0 +1,13 @@
+puts "Enter a number: "
+num = gets.chomp.to_i
+if num <= 1
+  puts "#{num} is not prime"
+else
+  for i in 2..num
```

Comment:
Can you check with some other way?
if number is 1000, loop will run 999 times.
Check for an optimised solution.

Reply:
An optimal solution is achieved by checking the divisibility of the number by any integer in the range from 2 to the square root of that number. This reduces search time.